dio/lib/src/sentry_dio_extension.dart

```diff
@@ -16,10 +17,20 @@ extension SentryDioExtension on Dio {
     bool recordBreadcrumbs = true,
     bool networkTracing = true,
     MaxRequestBodySize maxRequestBodySize = MaxRequestBodySize.never,
-    List<SentryStatusCode> failedRequestStatusCodes = const [],
```

Comment:
The reason for that is because `dio` itself already throws in some cases, right?

Reply:
Yes, exactly